rc/filetype/cmake.kak

```diff
@@ -19,12 +19,13 @@ provide-module cmake %{
 
 add-highlighter shared/cmake regions
 add-highlighter shared/cmake/code default-region group
-add-highlighter shared/cmake/comment  region '#' '$' fill comment
+add-highlighter shared/cmake/comment  region '\s*#' '$' fill comment
```

Comment:
That does not look necessary, the only effect of this would be to make the comment region start earlier but I dont expect any other regions to start in those whitespaces.

Reply:
you are right, fixed.